sufia-models/app/services/sufia/versioning_service.rb

```diff
@@ -0,0 +1,27 @@
+module Sufia
+  class VersioningService
+    # Make a version and record the version committer
+    # @param [ActiveFedora::File] content
+    # @param [User] user
+    def self.create(content, user=nil)
+      content.create_version
+      record_committer(content, user) if user
+    end
+
+    # @param [ActiveFedora::File] file
+    def self.latest_version_of(file)
+      file.versions.last
```

Comment:
@jcoyne I am seeing two ways of getting at the version file.content.versions.first and file.versions. Should these all be file.versions now, since content is implied, or is there a version for the file and a version for the content?